library/core/src/num/f64.rs

```diff
@@ -719,6 +719,62 @@ impl f64 {
         intrinsics::minnumf64(self, other)
     }
 
+    /// Returns the maximum of the two numbers.
+    ///
+    /// ```
+    /// #![feature(float_minimum_maximum)]
+    /// let x = 1.0_f64;
+    /// let y = 2.0_f64;
+    ///
+    /// assert_eq!(x.maximum(y), y);
+    /// assert!(x.maximum(f64::NAN).is_nan());
+    /// ```
+    ///
+    /// If one of the arguments is NaN, then NaN is returned. Otherwise this returns the greater
+    /// of the two numbers. For this operation, -0.0 is considered to be less than +0.0.
+    /// Note that this follow the semantics specified in IEEE 754-2019.
+    #[unstable(feature = "float_minimum_maximum", issue = "none")]
+    #[inline]
+    pub fn maximum(self, other: f64) -> f64 {
+        if self > other {
```

Comment:
I see that `min` uses `intrinsics::minnumf64` -- since it looks like there's a https://llvm.org/docs/LangRef.html#llvm-minimum-intrinsic, should this one also call an intrinsic?

(Though I guess that doesn't *have* to be this PR.)

Reply:
Oh, sorry, I missed that you mentioned this.